charts/stable/frigate/values.yaml

```diff
@@ -99,8 +99,10 @@ resources: {}
 # @default -- see URL to default config
 config: |
   mqtt:
-    host: test.mosquitto.org
+    host: mosquitto
     topic_prefix: frigate
+  database:
+    path: /data/frigate.db
```

Comment:
Will this work if there is no `/data` folder by default? It might make the CI / any first-time installs fail

Reply:
Hmm this is true, should I make a note in the custom config section of the readme?

Reply:
Yeah, that might be wise, and a `ct-values` file with an `emptyDir` is probably required.